app/models/design_file.rb

```diff
@@ -0,0 +1,4 @@
+# typed: strict
+class DesignFile < ApplicationRecord
+  has_many :text_layers, foreign_key: :design_file_id
+end
```

Comment:
The DesignFile model lacks validations for its attributes. It's important to validate the presence of attributes like 'name' to ensure data integrity. Consider adding validations such as 'validates :name, presence: true'.